tests/Field/DayOfMonthTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\DateTime\Tests\Field;
+
+use Brick\DateTime\Field\DayOfMonth;
+use Brick\DateTime\Tests\AbstractTestCase;
+
+/**
+ * Unit tests for class DayOfMonth.
+ *
+ * @doesNotPerformAssertions
```

Comment:
Does it work with the annotation on the method rather than on the class?
It would be better if we add other methods in the future.

Reply:
Yes. I set the wrong place for the annotation.